vcd/sample_vcd_test_config.json

```diff
@@ -11,13 +11,13 @@
     "allowInsecure": true,
     "//": "tfAcceptanceTests is needed to allow the whole test suite to run",
     "//": "Warning: it may cost time (from 14 to 40 min) and possibly money",
-    "tfAcceptanceTests": false,
+    "tfAcceptanceTests": true,
```

Comment:
Thanks! The only remaining question is what happens if we set this to false, but run `make testacc` which sets `TF_ACC=1` - which setting takes priority?

Reply:
if `TF_ACC` is set by `make`, it is used.
As it is now, this is a setting that can only be enabled, but not disabled. Having it in the configuration file is for convenience. 
I don't think it's a problem, because when the variable is not set the test is over in about 1 second and you can re-run with the appropriate setting.